stdlib/3/typing.pyi

```diff
@@ -338,28 +338,28 @@ class MappingView:
     def __len__(self) -> int: ...
 
 class ItemsView(MappingView, AbstractSet[Tuple[_KT_co, _VT_co]], Generic[_KT_co, _VT_co]):
-    def __and__(self, o: Iterable[_T]) -> AbstractSet[Union[Tuple[_KT_co, _VT_co], _T]]: ...
-    def __rand__(self, o: Iterable[_T]) -> AbstractSet[Union[Tuple[_KT_co, _VT_co], _T]]: ...
+    def __and__(self, o: Iterable[_T]) -> Set[Tuple[_KT_co, _VT_co]]: ...
```

Comment:
Since `_T` is only used once, it serves no purpose here. Following the [annotations for `set`](https://github.com/python/typeshed/blob/b94fa842eba642c73c25ec8cd33cd6bce1a8dc76/stdlib/2and3/builtins.pyi#L1007), I'd annotate it like this:

```suggestion
    def __and__(self, o: Iterable[object]) -> Set[Tuple[_KT_co, _VT_co]]: ...
```